core-ui/src/components/Predefined/Details/ServiceInstance/ServiceInstance.details.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { ServiceInstanceStatus } from 'shared/components/ServiceInstanceStatus';
+import { ServiceBindingList } from './ServiceBindingList';
+import { ServiceInstanceData } from './ServiceInstanceData';
+import { ReadonlyEditorPanel } from 'shared/components/ReadonlyEditorPanel';
+import { useTranslation } from 'react-i18next';
+
+export const ServiceInstancesDetails = ({
+  DefaultRenderer,
+  ...otherParams
+}) => {
+  const { t } = useTranslation();
+
+  const ServiceInstanceParameters = ({ spec }) => {
+    if (!spec.parameters) return;
+
+    return (
+      <ReadonlyEditorPanel
+        title={t('btp-instances.parameters')}
+        value={JSON.stringify(spec.parameters, null, 2)}
+        key="instance-parameters"
+        editorProps={{ language: 'json', height: '10em' }}
+      />
+    );
+  };
+
+  const customColumns = [
+    {
+      header: t('common.headers.status'),
+      value: resource => <ServiceInstanceStatus serviceInstance={resource} />,
+    },
+  ];
+
+  return (
+    <DefaultRenderer
+      customColumns={customColumns}
+      customComponents={[
+        ServiceInstanceData,
+        ServiceBindingList,
```

Comment:
We don't need the `service instance name` column on a service instance details view here. The user will know what instance it refers to.